internal/controllers/clusterextension_controller_test.go

```diff
@@ -223,7 +165,7 @@ func TestClusterExtensionUnpackUnexpectedState(t *testing.T) {
 	require.NoError(t, cl.DeleteAllOf(ctx, &ocv1alpha1.ClusterExtension{}))
 }
 
-func TestClusterExtensionResolutionAndUnpackSuccessfulApplierFails(t *testing.T) {
+func TestClusterExtensionUnpackSuccessfulApplierFails(t *testing.T) {
```

Comment:
Nit: I'd keep the original test name. Unpack being successful implies that resolution succeeds, but the context is helpful, IMO.

Reply:
Fixed https://github.com/operator-framework/operator-controller/commit/c214b0b405df178c7bb1ce9a5d9da31cec7ac332